x-pack/.i18nrc.json

```diff
@@ -18,7 +18,7 @@
     "xpack.idxMgmt": "legacy/plugins/index_management",
     "xpack.indexLifecycleMgmt": "legacy/plugins/index_lifecycle_management",
     "xpack.infra": "legacy/plugins/infra",
-    "xpack.kueryAutocomplete": "legacy/plugins/kuery_autocomplete",
+    "xpack.data": "plugins/data",
```

Comment:
Maybe we should also name the directory `data_enhanced` for consistency with the plugin IDs, WDYT @lizozom ?

It seems to me that if someone knows the name of the plugin as "data enhanced", they should look for a `data_enhanced` directory as well

Reply:
In this case almost all folders in /x-pack/plugins will have _enhanced postfix
We should follow one rule for naming.

Reply:
renamed